scss/tech.scss

```diff
@@ -1,43 +1,47 @@
 @import 'colors';
-@mixin round-container {
-  border-radius: 10px;
-  height: 410px;
-  width: 280px;
-}
-$background-offset: 20px;
 
-.content {
-  padding: 100px 150px;
+.layout {
+  display: grid;
+  grid-gap: 40px 0px;
+  grid-template-columns: 1fr auto 2fr;
+  grid-template-areas: 
+    "navbar navbar navbar"
+    "...... category ......"
+    "...... category ......"
+    "footer footer footer";
+  grid-template-rows: repeat(4,auto);
 }
 
-.background {
-  @include round-container;
-  border-color: $earls-green;
-  border-style: solid;
-  left: $background-offset;
-  position: absolute;
-  top: $background-offset;
-  transition: left 0.4s ease, top 0.4s ease;
-
-  &:hover {
-    left: 2 * $background-offset;
-    top: 2 * $background-offset;
-  }
+.category {
+  display: grid;
+  grid-area: category;
+  grid-row: span 1;
+  grid-template-areas: 
+    "title ....."
+    "..... techs";
+  grid-template-columns: repeat(2,auto);
+  grid-template-rows: repeat(2,auto);
 }
 
-.tech-container {
-  position: relative;
+.techs {
+  display: grid;
+  grid-area: techs;
+  grid-gap: 10px 10px;
+  grid-template-areas: 
+    "main other other"
+    "main other other";
```

Comment:
You shouldn't use the `template-area` in here, since it will be annoying when you add more techs. Just use the `grid-template-columns: repeat(3, max-content)`, CSS grid will know how to handle the positioning of the elements :smile_cat: